packages/teleport/src/components/TabIcon/index.ts

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
I think it's a broader topic but do we consider placing only reference to license file as eg Meta does? eg. https://github.com/facebook/react/blob/main/packages/react-devtools-core/src/standalone.js
What do you think?

Reply:
sure, i'm okay doing this if the others are okay with it